client/src/components/TrackList.vue

```diff
@@ -105,19 +102,6 @@ export default Vue.extend({
 
 
   methods: {
-    /**
-     * On load check allTypes to make sure it exists in the overall list if not default to unknown
-     * All changes should have a created type after initial load.
-     * Has the additional benefit if you delete the only newTrackSettings.type
-     * It will default back to the unknown type instead of creating a type that isn't known
-     */
-    checkExistingTrack(types: string[]) {
-      if (types.indexOf(this.newTrackSettings.value.type) === -1) {
-        const copy: NewTrackSettings = cloneDeep(this.newTrackSettings.value);
-        copy.type = 'unknown'; // Modify the value
-        this.$emit('update-new-track-settings', copy);
-      }
-    },
```

Comment:
Removal of this brings about a minor bug/issue.  If you do select a type from the dropdown for detections/tracks and then move to another folder/project it will load with the type as the default track.  This would swap it back to unknown if the type wasn't in the list of types.  I never really liked the way this was implemented but it provided a better user experience rather than having 'flounder' be the default track/detection type when you opened up a sea lion dataset because you worked on a fish one before.